crates/ruff_linter/src/rules/pylint/rules/repeated_keyword_argument.rs

```diff
@@ -56,9 +56,9 @@ pub(crate) fn repeated_keyword_argument(checker: &mut Checker, call: &ExprCall)
                     keyword.range(),
                 ));
             }
-        } else if let Expr::Dict(ExprDict { keys, .. }) = &keyword.value {
+        } else if let Expr::Dict(dict) = &keyword.value {
             // Ex) `func(**{"a": 1, "a": 2})`
-            for key in keys.iter().flatten() {
+            for key in dict.keys().into_iter().flatten() {
```

Comment:
```suggestion
            for key in dict.keys().iter().flatten() {
```

We should expose an `iter` function if it doesn't already exist (It's most common to have `iter` and `into_iter` but rare to onlyh have `into_iter`)